src/smt/proof_final_callback.h

```diff
@@ -74,6 +74,10 @@ class ProofFinalCallback : protected EnvObj, public ProofNodeUpdaterCallback
    * Counts number of theory ids in TRUST_THEORY_REWRITE steps.
    */
   HistogramStat<theory::TheoryId> d_trustTheoryIdCount;
```

Comment:
You could take the opportunity to rename the name of this variable, since it's a bit misleading now.